app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Great Quotes</h1>
\ No newline at end of file
+<h1>Great Quote</h1>
+<div class="container">
+  <div class="post">
+    <h4>Post <%= @post.title %></h4>
+    <p class="count">Comments: <%= @post.comments.count %> | Likes: <%= @post.likes.count %></p>
+    <p><%= @post.text %></p>
+  </div>
+
+  <div class="comments">
+    <% @post.comments.each do |comment| %>
+      <div class="comment">
+        <p><%= comment.author.name %>: <%= comment.text %></p>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] Great effort so far 👏✨ However, I noticed a small issue in the implementation. While I can navigate to the post show page from the user show page for a post without comments, I encountered an error when clicking on a post with comments 🐛⚠ Please refer to the following video for better clarity 👇 Kindly ensure to address this for a smoother user experience.


_Due to this bug, I am unable to navigate through the post index and post show pages. Please ensure that it is functioning correctly so that the next reviewer can review it_

https://github.com/evansnyamekye/blog-app/assets/106119105/d4062f2b-b482-4f81-872a-96b257822206